docs/setup.md

```diff
@@ -1,57 +1,143 @@
+The configuration settings of the **Record Distribution** Widget is slightly different for dashboards or reports and the detail view of a module's detail view. The reason is that in the case of dashboards or reports, you can choose any module and view its records based on the specified grouping. However, in the case of the detail view of a module's record, you can choose only the related modules.
+
+Following table helps customize the **Record Distribution** widget:
+
+| Fields                                     | Description                                                                                                                               |
+|--------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| Title                                      | Specify a title for the visual depiction of each record node in the group.                                                                |
+| Select Data Source (Dashboards or Reports) | Select the module whose records to group in a *Dashboard* or A *Report*.                                                                  |
+| Select Related Data Source (View Panels)   | Select the related module whose records to group in *View Panels*.                                                                        |
+| Picklist                                   | Select the picklist using which to group the records in the selected module.                                                              |
+| Picklist Items                             | Choose the picklist items to be displayed by the widget.                                                                                  |
+| Icon for Record View                       | Select a field of type `Lookup` using which to display the icons representing asset records. For more information, refer to [set up modules to use icons that represent records](#setting-up-modules-to-use-icons-for-records). |
+| Show Correlation Edges                     | Select this option to add visual lines on the view to depict correlations between records from the same module grouped at various levels. |
+| Record Assignment (Default Filter)         | Select to display records that are assigned to the logged-in user or to display all records based on the defined filters.                 |
+| Assignment Fields                          | Select the field based on which records are assigned for the select module.                                                               |
+| Filter Criteria                            | Define the filter criteria using which to filter the data retrieved by this widget.                                                       |
+
```

Comment:
One of the field "Title for Record View" is missing

Reply:
![image](https://github.com/fortinet-fortisoar/widget-record-distribution/assets/100339208/c720f921-dc4d-43e0-baa1-ef2c6f7feb27)

@cs-ankit-gupta: Please provide inputs.

Reply:
@cs-abhishek-shukla This field represents the title displayed below each icon in the Record distribution widget. Users can configure which title widget can show, like Name, IP address, etc.